invenio_app_ils/config.py

```diff
@@ -665,6 +669,37 @@ def _(x):
         update_permission_factory_imp=deny_all,
         delete_permission_factory_imp=deny_all,
     ),
+    fssid=dict(
+        pid_type=FRONT_SITE_SEARCH_PID_TYPE,
+        pid_minter=FRONT_SITE_SEARCH_PID_MINTER,
+        pid_fetcher=FRONT_SITE_SEARCH_PID_FETCHER,
+        search_class=FrontSiteSearch,
+        record_serializers={
+            "application/json": (
+                "invenio_app_ils.records.serializers:json_v1_response"
+            )
+        },
+        search_serializers={
+            "application/json": (
+                "invenio_app_ils.records.serializers:json_v1_search"
+            ),
+            "text/csv": ("invenio_app_ils.records.serializers:csv_v1_search"),
+        },
+        search_serializers_aliases={
+            "csv": "text/csv",
+            "json": "application/json",
+        },
+        item_route="/frontsite-search/<pid(fssid):pid_value>",
+        list_route="/frontsite-search/",
+        default_media_type="application/json",
+        max_result_window=_RECORDS_REST_MAX_RESULT_WINDOW,
+        error_handlers=dict(),
+        list_permission_factory_imp=allow_all,
```

Comment:
You will still have to take into account closed access docs: https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/documents/search.py#L24